src/data/languages.json

```diff
@@ -13,6 +13,7 @@
 		{ "language": "diff", "image": "manifest" },
 		{ "language": "dockerfile", "image": "docker" },
 		{ "language": "fsharp", "image": "fsharp" },
+		{ "language": "fortran", "image" : "fortran"},
```

Comment:
Could you match the spacing at the end bracket with the one on the starting one?